psp/__init__.py

```diff
@@ -1,7 +1,8 @@
-__all__ = ['Pv','options']
+__all__ = ['Pv', 'options']
 
-from psp.Pv import Pv as PV
+from psp.Pv import Pv as PV  # noqa
```

Comment:
This file has (always?) had a desync between the `__all__` and the import here

Reply:
I had assumed some other dependency had `from psp import PV` and didn't want to break that.

Reply:
But `__all__` should have `PV` instead of `Pv`, right?

Reply:
Or does it as written give you the entire submodule on the `*` import anyway?

Reply:
TIL

Reply:
Nothing to change here, I'm not well-versed in star imports apparently